src/anyvar/utils/liftover_utils.py

```diff
@@ -0,0 +1,292 @@
+
+        - `AccessionConversionError`: If unable to lift over the variant's refget accession
+    """
+    if not variant_object:
+        raise MalformedInputError
+
+    # Get variant start position, end position, and refget accession - liftover is currently unsupported without these
+    refget_accession = (
+        variant_object.get("location", {})
+        .get("sequenceReference", {})
+        .get("refgetAccession")
+    )
+    start_position = variant_object.get("location", {}).get("start")
+    end_position = variant_object.get("location", {}).get("end")
+    if not refget_accession or not start_position or not end_position:
+        raise UnsupportedVariantLocationTypeError
+
+    # Determine which assembly we're converting from/to
+    prefixed_accession = f"ga4gh:{refget_accession}"
+
+    grch37 = "GRCh37"
+    grch38 = "GRCh38"
+
+    # This is required for `get_from_and_to_assemblies` and `get_chromosome_from_aliases`
+    # See function documentation for more details
+    accession_aliases = {
+        grch37: list(
+            seqrepo_dataproxy.translate_sequence_identifier(prefixed_accession, grch37)
+        ),
+        grch38: list(
+            seqrepo_dataproxy.translate_sequence_identifier(prefixed_accession, grch38)
+        ),
+    }
+
+    from_assembly = None
+    to_assembly = None
+    from_assembly, to_assembly = get_from_and_to_assemblies(
+        accession_aliases
+    )  # Will raise an `UnsupportedReferenceAssemblyError` or `AmbiguousReferenceAssemblyError` if unsuccessful
+
+    # Determine which chromosome the variant is on
+    chromosome = get_chromosome_from_aliases(accession_aliases.get(from_assembly, []))
+    if not chromosome:
+        raise ChromosomeResolutionError
+
+    # Begin liftover conversion
+    assembly_map = {grch37: Genome.HG19, grch38: Genome.HG38}
+    converter = Converter(assembly_map[from_assembly], assembly_map[to_assembly])
+
+    # Get converted start/end positions. `convert_position` will raise a `CoordinateConversionError` if unsuccessful
+    converted_start = convert_position(converter, chromosome, start_position)
+    converted_end = convert_position(converter, chromosome, end_position)
+
+    # Get converted refget_accession (without 'ga4gh:' prefix)
+    new_alias = f"{to_assembly}:{chromosome}"
+    converted_refget_accession = seqrepo_dataproxy.translate_sequence_identifier(
+        new_alias, "ga4gh"
+    )[0].split("ga4gh:")[1]
+    if not converted_refget_accession:
+        raise AccessionConversionError
+
+    # Build the converted location dict
+    converted_variant_location = {
+        "start": converted_start,
+        "end": converted_end,
+        "id": None,
+        "sequenceReference": {
+            "type": "SequenceReference",
+            "refgetAccession": converted_refget_accession,
+        },
+    }
+
+    # Build the liftover variant object
+    # Start by copying the original variant
+    converted_variant_dict = variant_object
```

Comment:
fwiw this isn't a copy, these will both point to the same object in memory. Is there a reason we need to share any data between the old + new variants?